app/utils/checkOrigin.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const {parse} = require('url');
```

Comment:
As per previous comment...
```suggestion
const { parse } = require('url');
```